about/list_of_features.rst

```diff
@@ -0,0 +1,514 @@
+- Screen-space ambient occlusion at half or full resolution.
+- Glow/bloom with optional bicubic upscaling and several blend modes available:
+  Screen, Soft Light, Add, Replace, Mix.
+- Color correction using an one-dimensional ramp.
+- Roughness limiter to reduce the impact of specular aliasing.
+- Brightness, contrast and saturation adjustments.
+
+**Texture filtering:**
+
+- Nearest, bilinear, trilinear or anisotropic filtering.
+- Filtering options are defined on a per-use basis, not a per-texture basis.
+
+**Texture compression:**
+
+- Basis Universal (slow, but results in smaller files).
+- BPTC for high-quality compression (not supported on macOS).
+- ETC2 (not supported on macOS).
+- S3TC (not supported on mobile/Web platforms).
+
+**Anti-aliasing:**
+
+- Fast approximate antialiasing (FXAA).
+- Multi-sample antialiasing (MSAA).
+
+Most of these effects can be adjusted for better performance or to further
+improve quality. This can be helpful when using Godot for offline rendering.
+
+3D tools
+^^^^^^^^
+
+- Built-in meshes: cube, cylinder/cone, (hemi)sphere, prism, plane, quad.
+- Tools for :ref:`procedural geometry generation <doc_procedural_geometry>`.
+- :ref:`Constructive solid geometry <doc_csg_tools>` (intended for prototyping).
+- Path3D node to represent a path in 3D space.
+   - Can be drawn in the editor or generated procedurally.
+   - PathFollow3D node to make nodes follow a Path3D.
+- 3D geometry helper class.
+
+3D physics
+^^^^^^^^^^
+
+**Physics bodies:**
+
+- Static bodies.
+- Rigid bodies.
+- Kinematic bodies.
+- Vehicle bodies (intended for arcade physics, not simulation).
+- Joints.
+- Soft bodies.
+- Ragdolls.
+- Areas to detect bodies entering or leaving it.
+
+**Collision detection:**
+
+- Built-in shapes: infinite plane, cuboid, sphere, capsule, cylinder (only with
+  Bullet physics).
+- Generate triangle collision shapes for any mesh from the editor.
+- Generate one or several convex collision shapes for any mesh from the editor.
+
+Shaders
+^^^^^^^
+
+- *2D:* Custom vertex, fragment, and light shaders.
+- *3D:* Custom vertex, fragment, light, and sky shaders.
+- Text-based shaders using a `shader language inspired by GLSL <doc_shading_language>`.
+- Visual shader editor.
+   - Support for visual shader plugins.
+
+Scripting
+^^^^^^^^^
+
+**General:**
+
+- Signals and groups.
+- Many built-in datatypes to ease the development process:
+   - 2D and 3D vectors, available in floating-point and integer form.
+   - 2D and 3D transforms.
+   - 2D rectangles.
+   - 3D axis-aligned bounding boxes (AABBs).
+   - 3D infinite planes.
+   - Colors.
+   - Quaternions.
+
+**GDScript:**
+
+- :ref:`High-level interpreted language <doc_gdscript>` with
+  :ref:`optional static typing <doc_gdscript_static_typing>`.
+- Syntax inspired by Python.
+- Syntax highlighting is provided on GitHub.
+- :ref:`Use threads <doc_using_multiple_threads>` to perform asynchronous actions
+  or make use of multiple processor cores.
+
+**C#:**
+
```

Comment:
I still think we should link articles, but now I'm thinking that we should do this for all languages:

https://docs.godotengine.org/en/latest/getting_started/scripting/c_sharp/index.html

https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/index.html

https://docs.godotengine.org/en/latest/getting_started/scripting/visual_script/index.html